Taskfile.yml

```diff
@@ -115,3 +115,17 @@ tasks:
         sh: 'cat assets/Zeugnis.pdf | {{if eq OS "darwin"}}base64{{else}}base64 -w 0{{end}}'
     requires:
       vars: [STUDENT_ID]
+
+  audit-log:
+    desc: Get the audit log for a student
+    cmds:
+      - "http -b GET localhost:8099/students/{{.STUDENT_ID}}/log 'X-API-KEY: This_is_a_test_APIKEY_with_30_chars+' 'Accept: test/plain'"
+    requires:
+      vars: [STUDENT_ID]
+
+  audit-log-json:
+    desc: Get the audit log for a student
+    cmds:
+      - "http -b GET localhost:8099/students/{{.STUDENT_ID}}/log?verbose=true 'X-API-KEY: This_is_a_test_APIKEY_with_30_chars+' 'Accept: application/json' | jq"
```

Comment:
Afaik you shouldn't say `verbose=true`, but rather just `verbose`. And the absence of the parameter means that it is false. I'm not sure though if this works with typescript-rest. But I would expect it to.

Reply:
it does not :(

Reply:
We should add support for it in our fork.

Reply:
empty string means true, right?